content/blog/announce-tonic-0-1.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "Tonic: gRPC has come to async/await!"
+url: "/tonic-grpc-has-come-to-async-await"
+date: 2019-10-01T10:00:00-04:00
+draft: false
+---
+
+# Tonic: gRPC has come to async/await!
+
+I am pleased to finally announce a crate that I have been working very hard on for the
+past few months. [`tonic`] has finally hit the initial [`0.1.0-alpha.1`] release! Stable
+releases will follow in the coming months.
+
+### What is Tonic?
+
+> Tonic is a gRPC over HTTP/2 implementation focused on high performance, interoperability, and flexibility. This library was created to have first class support of async/await and to act as a core building block for production systems written in Rust.
+
+Tonic's prior incarnation was known as [`tower-grpc`]. It was built to satisfy [linkerd]'s need for a production ready gRPC implementation. [`tower-grpc`] was based upon the [`tower`] library, which has production uses in systems such as the [`linkerd-proxy`], [`vector`], and the [`noria`] database at MIT. The [`tower`] library has also seen usage in additional libraries such as  [`warp`], [`tower-web`], and [`actix-net`].
+
+With async/await’s forthcoming stabilization, we created Tonic to support the new syntax natively. This means clients out of the box will support async/await and server
+implementations can be defined via [`async_trait`]'s. This provides an unparalleled experience for
+writing async services quickly and efficiently. Not only does Tonic provide a fully featured gRPC implementation but it also comes with a fully featured, batteries included, HTTP/2 client and server built
```

Comment:
I think this is normally hyphenated?
```suggestion
writing async services quickly and efficiently. Not only does Tonic provide a fully featured gRPC implementation but it also comes with a fully featured, batteries-included, HTTP/2 client and server built
```

Reply:
Also, you say "fully-featured" twice in this sentence. Maybe change one of them?